TTHAnalysis/cfg/run_susyMultilepton_cfg.py

```diff
@@ -44,8 +44,14 @@
 # Lepton Skimming
 ttHLepSkim.minLeptons = 2
 ttHLepSkim.maxLeptons = 999
-#ttHLepSkim.idCut  = ""
-#ttHLepSkim.ptCuts = []
+ttHLepSkim.allowLepTauComb = True
+
+if analysis=='susy':
+    susyCoreSequence.remove(ttHLepSkim)
+    susyCoreSequence.insert(susyCoreSequence.index(ttHLepSkim)+1,globalSkim)
```

Comment:
First ttHLepSkim, then one looks for where ttHLepSkim is. It should be the opposite.